chat-log/chatlog.go

```diff
@@ -0,0 +1,33 @@
+package chatlog
+
+import "github.com/Sirupsen/logrus"
+
+var log = logrus.New()
+
+type Fields map[string]interface{}
+
+func init() {
+
+	log.Formatter = &logrus.JSONFormatter{}
+}
+
+func Log(fields map[string]interface{}, log_type string, message ...interface{}) {
+	switch log_type {
+	case "error":
+		log.WithFields(fields).Error(message)
+	case "warning":
+		log.WithFields(fields).Warning(message)
+	case "fatal":
+		log.WithFields(fields).Fatal(message)
+	default:
+		log.WithFields(fields).Info(message)
+	}
+}
+
+func HandleError(err error, error_message string){
```

Comment:
This method should return error or bool
Here some [example](https://github.com/8tomat8/webAppSkeleton/blob/master/environment/helpers.go) how it could be done.